docker/dockerfile.merlin

```diff
@@ -167,12 +167,12 @@ CMD ["/bin/bash"]
 
 FROM ${BASE_IMAGE} as base-amd64
 ONBUILD COPY --chown=1000:1000 --from=triton /usr/lib/x86_64-linux-gnu/libdcgm.so.2 /usr/lib/x86_64-linux-gnu/libdcgm.so.2
-ONBUILD COPY --chown=1000:1000 --from=triton /usr/local/cuda-11.8/targets/x86_64-linux/lib/libcupti.so.11.8 /usr/local/cuda-11.8/targets/x86_64-linux/lib/libcupti.so.11.8
```

Comment:
@jperez999 Are we ready to make this switch from 11.8 to 12.0?

Reply:
I get the following error trying to build this locally. Which `TRITON_VERSION` and `DLFW_VERSION` are you building with?

```
Step 72/165 : FROM base-${TARGETARCH} as base
# Executing 3 build triggers
COPY failed: stat usr/local/cuda-12.0/targets/x86_64-linux/lib/libcupti.so.12: file does not exist
```

Reply:
Hi~ Thanks for your review, the I used 23.02 for TRITON_VERSION and DLFW_VERSION. BTW, will we upgrade upstream to 23.03 in our 23.04 release?

Reply:
@jperez999 Thoughts on which upstream versions we'll use?